.github/workflows/check_files_submitted.py

```diff
@@ -0,0 +1,88 @@
+"""
+Script to check if the number of files submitted in a Pull Request exceeds 20.
+
+Methodology:
+    Analyses the text file generated after the git diff command.
+    Checks the number of files modified in the PR branch by
+    counting the lines in the file.
+    Exits with an appropriate error message if the number of files exceeds 20.
+
+NOTE:
+    This script complies with Python 3 coding and documentation standards,
+    including:
+        1) Pylint
+        2) Pydocstyle
+        3) Pycodestyle
+        4) Flake8
+    Run these commands from the CLI to ensure the code is compliant
+    for all your pull requests.
+"""
+
+import sys
+import subprocess
+
+
+def check_file_lines(file_name):
+    """
+    Check the number of lines in a file.
+
+    Args:
+        file_name (str): Name of the file to check.
+
+    Returns:
+        bool: True if the number of lines is within limit, False otherwise.
+    """
+    try:
+        with open(file_name, "r", encoding="utf-8") as file:
+            num_lines = len(file.readlines())
+            if num_lines > 20:
+                print("Number of changed files exceeds 20.")
+                return False
+    except FileNotFoundError:
+        print(f"File '{file_name}' not found.")
+        return False
+
+    return True
+
+
+def run_checks():
```

Comment:
This isn't required. The formatting requirement was for your benefit. It wasn't required for the script to police itself.